### ISSUE TITLE
Failed to import YouTube

### ISSUE THREAD

**Issue by purnendushukla** [pytube/pytube#260]
pytube version 9.2.2
Every time I try to use the PYTUBE module it shows this error.

from pytube import YouTube
ImportError: cannot import name 'YouTube'

installed pytube using pip
pip install pytube

**Comment by vn-ki** [comment]
Is this still a problem? Did you manage to solve it?

**Comment by purnendushukla** [comment]
problem already solved.
Thank you.

**Comment by Jumas-Cola** [comment]
Сказал бы хоть, каким образом она решилась.

**Comment by EllainYang** [comment]
For my case the decision was reinstall pytube and delete .pyc files (Linux(ubuntu)) (find . -name '*.pyc' -delete)

**Comment by CtrlAlt-Delete** [comment]
Could you elaborate on how you fixed it? I'm having the same problems. I'm not really sure what you mean by "(find . -name '*.pyc' -delete)"

**Comment by mikejpoggi** [comment]
Hello, I also had this error, and fixed it using EllainYang's comment. Just to clarify for anyone in the future who has this issue.
1. I uninstalled pytube3 from command line: >pip uninstall pytube3
2. I reinstalled pytube3 from command line: >pip install pytube3
3. Step 2 returns the filepath where pytube3 was installed. I went to the folder that was returned, opened "pytube/", and cleared the contents of the '\_\_pycache__' folder, which has all the ".pyc" files. These are not important and can be regenerated by the package whenever it is loaded.
4. I opened python and successfully ran: >>>from pytube import YouTube
Don't forget to capitalize the T!
